README.md

```diff
@@ -5,11 +5,27 @@ A digital repository application in use at Princeton University Library for stor
 [![CircleCI](https://circleci.com/gh/pulibrary/figgy.svg?style=svg)](https://circleci.com/gh/pulibrary/figgy)
 [![Browserstack](./browserstack-logo.svg)](https://www.browserstack.com/)
 
-## Dependencies
+## Language Dependencies
+
+Figgy provides a `.tool-versions` file -- consule this file for required languages and their current versions. `asdf` users should ensure all plugins listed there are installed via `asdf plugin-add` and then run `asdf install`.
+
```

Comment:
asdf plugin-add -> usage: asdf plugin-add <name> [<git-url>]
what should I run to install java zulu-8.56.0.21 ?

Reply:
ok I ran `asdf plugin-add java`

Reply:
Do you need to add this ^^ in the Java via ASDF on Mac section?

Reply:
I can make this more explicit. It's not related to the Mac business.

Reply:
@christinach updated.